db/migrations/mysql/000076_upgrade_lastrootpostat.up.sql

```diff
@@ -0,0 +1,23 @@
+ALTER TABLE Channels ALTER COLUMN LastRootPostAt SET DEFAULT 0;
+
+-- fixes migrate cte and sets the LastRootPostAt for channels that don't have it set
+UPDATE
```

Comment:
Do we have a way of understanding if this migration needs to be done? If so, it would be great to go defensive here too.

Reply:
I think the condition would be, do we have any `LastRootPostAt` that is `NULL` if so run the migration